doc/releases/changelog-0.36.0.md

```diff
@@ -551,6 +551,10 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* Using shot vectors with `param_shift(... broadcast=True)` caused a bug. This combination is no longer supported
```

Comment:
Should this be phrased as a bug fix or a breaking change?

Reply:
Fair point! It "fixes" a silent bug. And it is a breaking change in that wrong results are caught rather than returned to the user :thinking:

Reply:
I would definitely call this more a bug fix, since we are stopping bad behavior.